src/Symfony/Component/Ldap/Entry.php

```diff
@@ -38,13 +45,20 @@ public function getDn()
     /**
      * Returns whether an attribute exists.
      *
-     * @param string $name The name of the attribute
+     * @param string $name          The name of the attribute
+     * @param bool   $caseSensitive Whether the check should be case-sensitive
      *
      * @return bool
      */
-    public function hasAttribute($name)
+    public function hasAttribute($name, $caseSensitive = true)
     {
```

Comment:
```suggestion
    public function hasAttribute($name /* , $caseSensitive = true */)
    {
        $caseSensitive = 2 > \func_num_args() || true === \func_get_arg(1);
```

Adding a parameter to the signature of a public method is a BC break. You can document and use it already, but we have to wait until Symfony 6 to add it to the signature.

Reply:
Is the Symfony code style to comment out the parameter like you have in the suggestion? Or should I remove it entirely?

Reply:
Yes, the comment reminds us to enable the parameter on the next major version.